optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/StatisticUtils.java

```diff
@@ -76,10 +76,24 @@ public static String getStandardDeviationString(double[] standardDeviationDouble
             // Abbreviate to 2 decimals
             // We don't use DecimalFormat to abbreviate because it's written locale insensitive (like java literals)
             int dotIndex = abbreviated.lastIndexOf('.');
-            if (dotIndex >= 0 && dotIndex + 3 < abbreviated.length()) {
-                abbreviated = abbreviated.substring(0, dotIndex + 3);
+            StringBuilder result = new StringBuilder().append(abbreviated.substring(0, dotIndex < 0 ? abbreviated.length() : dotIndex));
+            if (dotIndex >= 0) {
+                char[] abbreviatedChars = abbreviated.toCharArray();
+                for (int i = 0; i < 3; i++) {
+                    if (dotIndex + i >= abbreviatedChars.length) {
+                        break;
+                    }
+                    char currentChar = abbreviatedChars[dotIndex + i];
+                    if (Character.isDigit(currentChar) || currentChar == '.') {
+                        result = result.append(currentChar);
+                    }
+                }
+                int expIndex = abbreviated.lastIndexOf('E');
+                if (expIndex >= 0) {
+                    result = result.append(abbreviated.substring(expIndex));
```

Comment:
no need to put the append return value back into the result. it's always the same instance with StringBuilder (unlike String and BigDecimal). The reason they return this is simply for method chaining: result.append("i = ").append(i);